sdk/src/test/java/co/decodable/sdk/pipeline/PurchaseOrder.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Copyright Decodable, Inc.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package co.decodable.sdk.pipeline;
+
+import org.apache.flink.shaded.jackson2.com.fasterxml.jackson.annotation.JsonProperty;
```

Comment:
I can't describe how much I hate the fact that Flink shades Jackson ;)

Reply:
You don't have to use Flink's Jackson version. Just add it as a dependency. It is generally a bad idea to rely on Flink shaded stuff, as there's no guarantee that Flink keeps those dependencies across versions.

Reply:
Ah ok, you are using Flink's `JsonDeserializationSchema`.

Reply:
Yes, exactly.